src/hooks/use-outside-of-list-intersection.hook.ts

```diff
@@ -44,8 +44,8 @@ export const useOutsideOfListIntersection = (
     };
 
     if (element && !parentRef) {
-      element.measureInWindow((x, y, width, height) => {
-        handleNewDimensions(Dimensions.get('window'), { x, y, width, height });
+      element.measure((x, y, width, height) => {
```

Comment:
Please check how this hook works on 'Wallet' screen now. If everything is OK, just resolve this conversation.

Reply:
but there is a different hook here `useListElementIntersection`